radis/misc/profiler.py

```diff
@@ -105,10 +105,16 @@ def percentage_distribution(self):
             # Adding percentage of time taken by each key
             for i in range(1, upper_limit):
                 for j in verbose_distribution[i]:
-                    verbose_distribution[i][j].insert(
-                        0,
-                        verbose_distribution[i][j][0] / total_sum_verbose_wise[i] * 100,
-                    )
+                    try:
+                        verbose_distribution[i][j].insert(
+                            0,
+                            verbose_distribution[i][j][0]
+                            / total_sum_verbose_wise[i]
+                            * 100,
+                        )
+                    except:
```

Comment:
Good practice : can  you always precise the excepted error ? (KeyError probably here)?
Having an open except comment makes the code harder to read.

Reply:
agreed